scripts/init-database.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import yaml
+import sqlalchemy
+import os
+import getpass
+
+configs = 'configurations'
+path = 'root_path'
+
+
+def retreive_values():
+    global path
+    global configs
+    path= os.path.abspath(os.path.join(os.path.join(__file__,os.pardir),os.pardir))
+    stream = open(path+"/bassa.yml", "r")
+    configs = yaml.load(stream)
+
+
+def create_database():
+    try:
+        os.environ['MYSQL_ROOT_PASSWORD']
+    except:
+        root_password = getpass.getpass(prompt = 'Enter root password:')
+        os.environ['MYSQL_ROOT_PASSWORD'] = root_password
+    connection_url = configs['database']['database_type']+'://root:'+os.environ['MYSQL_ROOT_PASSWORD']+'@'+configs['database']['database_ip']
+    engine = sqlalchemy.create_engine(connection_url)
+    engine.execute("CREATE DATABASE IF NOT EXISTS " + configs['database']['database_name'])
+    engine.execute("CREATE USER "+configs['database']['database_user_username']+"@"+configs['database']['database_ip']+" IDENTIFIED BY " + configs['database']['database_user_password'])
+    engine.execute("GRANT INSERT, UPDATE, SELECT, DELETE ON "+configs['database']['database_name']+".* TO "+configs['database']['database_user_username']+"@"+configs['database']['database_ip'])
+    engine.execute("FLUSH PRIVILEGES")
```

Comment:
Since you are granting permissions using `GRANT` you really don't need to `FLUSH PRIVILEGES`. But that is okay.